pontoon/settings/base.py

```diff
@@ -301,11 +301,16 @@ def _default_from_email():
             log = logging.getLogger(__name__)
             log.error(f"Invalid IP or IP range defined in BLOCKED_IPS: {ip}")
 
+THROTTLE_MAX_COUNT = 100  # A maximum of 100 requests
+THROTTLE_PERIOD = 60  # allowed in 1 minute.
+THROTTLE_BLOCK_DURATION = 600  # Subsequent requests are blocked for 10 minutes.
+
 MIDDLEWARE = (
     "django.middleware.security.SecurityMiddleware",
     "whitenoise.middleware.WhiteNoiseMiddleware",
     "django.middleware.gzip.GZipMiddleware",
     "pontoon.base.middleware.RaygunExceptionMiddleware",
+    "pontoon.base.middleware.ThrottleIpMiddleware",
```

Comment:
Should we have a setting, e.g. THROTTLE_ENABLED, and include this conditionally?